cmd/main.go

```diff
@@ -14,18 +14,27 @@ import (
 
 func main() {
 	logr.Infoln("Grabbing configuration...")
-	config, err := config.GetConfig()
-	if err != nil {
-		logr.Fatalf("Failed to get configuration: %v", err)
-	}
+	config := config.GetConfig()
 	logr.SetLevel(logr.Level(config.LogLevel))
 
-	names := gitapi.GetList(config)
-	gogit.UpdateLocalCopies(names, config)
+	orgaRepoNames := gitapi.GetRepoList(config, nil)
+	logr.Info("Found ", len(orgaRepoNames), " repositories in the organization")
+
+	gogit.UpdateLocalCopies(orgaRepoNames, config, nil)
+
+	// loop through the users in the config and log each users name to the console
+	if config.CloneUserRepos {
+		for _, user := range config.Users {
+			logr.Printf("[API] Found user: %v", user)
+			userRepoNames := gitapi.GetRepoList(config, &user)
+			logr.Info("Found ", len(userRepoNames), " repositories on the user account of ", user.Name)
+			gogit.UpdateLocalCopies(userRepoNames, config, &user)
+		}
+	}
 
 	UpdateInterval := cron.New()
 	UpdateInterval.AddFunc("*/3 * * * *", func() {
```

Comment:
The update interval should be configurable

Reply:
fixed in commit ddf71a5